ResourceManager/GitExtensionsControl.cs

```diff
@@ -123,7 +123,7 @@ public static bool IsTextEditKey(Keys keys, bool multiLine = false)
             keys &= ~Keys.Shift; // ignore the SHIFT key as modifier
             switch (keys)
             {
-                case Keys key when (key >= Keys.A && key <= Keys.Z) || (key >= Keys.D0 && key <= Keys.D9) || (key >= Keys.Oem1 && key <= Keys.Oem102):
+                case Keys key when key is >= Keys.A and <= Keys.Z or >= Keys.D0 and <= Keys.D9 or >= Keys.Oem1 and <= Keys.Oem102:
```

Comment:
How about this?
```diff
diff --git a/ResourceManager/GitExtensionsControl.cs b/ResourceManager/GitExtensionsControl.cs
index 2c23fc061..f2301ec8c 100644
--- a/ResourceManager/GitExtensionsControl.cs
+++ b/ResourceManager/GitExtensionsControl.cs
@@ -123,7 +123,10 @@ public static bool IsTextEditKey(Keys keys, bool multiLine = false)
             keys &= ~Keys.Shift; // ignore the SHIFT key as modifier
             switch (keys)
             {
-                case Keys key when key is >= Keys.A and <= Keys.Z or >= Keys.D0 and <= Keys.D9 or >= Keys.Oem1 and <= Keys.Oem102:
+                case Keys key when key is
+                    (>= Keys.A and <= Keys.Z) or
+                    (>= Keys.D0 and <= Keys.D9) or
+                    (>= Keys.Oem1 and <= Keys.Oem102):
                 case Keys.Space:
                 case Keys.Back:
                 case Keys.Delete:
```

Reply:
Considering what the code on the left is, I'm fine with keeping one line as long as the parentheses are returned. If wrapping is applied, I would suggest placing the `or` operators at the beginning of lines instead of the end.

Reply:
> suggest placing the `or` operators at the beginning of lines instead of the end.

Works for me